### ISSUE TITLE
with_cp confusing

### ISSUE THREAD

**Issue by torinchen** [facebookresearch/HRViT#5]
why with_cp flag is set, the norm's momentum need reset?
if with_cp:
    momentum = 1 - 0.9 ** 0.5
    self.norm_cfg.update(dict(momentum=momentum))

**Comment by JeremieMelo** [comment]
https://discuss.pytorch.org/t/checkpoint-with-batchnorm-running-averages/17738
